x-pack/legacy/plugins/code/server/distributed/code_services.test.ts

```diff
@@ -17,12 +17,12 @@ import { Logger } from '../log';
 import { ConsoleLoggerFactory } from '../utils/console_logger_factory';
 
 const log: Logger = new ConsoleLoggerFactory().getLogger(['test']);
-let hapiServer: Server = createTestHapiServer();
+let hapiServer = createTestHapiServer();
```

Comment:
@restrry any util functions to create the router for unit tests?

Reply:
you can create a mocked instance of the router 
```js
import { httpServiceMock } from 'src/core/server/mocks';
const routerMock = httpServiceMock.createRouter();
```